test/hub/unit/MultiShareClass.t.sol

```diff
@@ -327,17 +327,26 @@ contract ShareClassManagerSimpleTest is ShareClassManagerBaseTest {
         assertEq(shareClass.previewShareClassId(poolId, index).raw(), bytes16((uint128(poolId.raw()) << 64) + index));
     }
 
+    function testUpdateShareClass() public {
```

Comment:
NIT. A leftover from a previous refactor. Could you rename this test file to `ShareClassManager.t.sol`? I can do it in a follow-up PR if not.